modules/cnf-image-based-upgrade-seed-image-config.adoc

```diff
@@ -140,6 +144,40 @@ a|`SriovNetwork.yaml`
 `SriovNetworkNodePolicy.yaml`
 |If the configuration, including namespaces, is exactly the same on both the seed and target cluster, you can include them in the seed image. Otherwise, apply them as extra manifests.
 |====
+endif::[]
+
+ifdef::ibi[]
+The following table lists resources and configurations that you must apply as extra manifests or by using {rh-rhacm} policies:
+
+.Additional resources for image-based installations
+[cols=2*, width="80%", options="header"]
+|====
+|Resource
+|Application
+
+|`ClusterLogForwarder.yaml`
+|Extra manifest
+
+|`ReduceMonitoringFootprint.yaml`
+|Extra manifest
+
+|`SriovFecClusterConfig.yaml`
+|Extra manifest
+
+|`PtpOperatorConfigForEvent.yaml`
+|Extra manifest
+
+|`DefaultCatsrc.yaml`
+|Extra manifest
+
+|`PtpConfig.yaml`
+|If the interfaces of the target cluster are common with the seed cluster, you can apply them through {rh-rhacm} policies. Otherwise, apply it as extra manifests.
+
+a|`SriovNetwork.yaml`
+`SriovNetworkNodePolicy.yaml`
+|If the configuration, including namespaces, is exactly the same on both the seed and target cluster, you can apply them through {rh-rhacm} policies. Otherwise, apply them as extra manifests.
```

Comment:
Ditto, these are not created on the seed cluster so I think they can be created by either extra manifests or ACM policies.